src/runner.ts

```diff
@@ -225,13 +225,18 @@ export class ActionRunner {
           usersToAdd = [...new Set([...usersToAdd, ...members])];
         }
       }
-      requirements.push({ users: usersToAdd, requiredApprovals: reviewers.min_approvals });
+      requirements.push({
+        users: usersToAdd,
+        requiredApprovals: reviewers.min_approvals,
+        countAuthor: reviewers.countAuthor ?? false,
+      });
     }
 
     // We count how many reviews are needed in total
     const requiredAmountOfReviews = rule.reviewers.map((r) => r.min_approvals).reduce((a, b) => a + b, 0);
-    // We get the list of users that approved the PR
-    const approvals = await this.prApi.listApprovedReviewsAuthors();
+    // We get the list of users that approved the PR and the author
+    // we filter the author later
+    const approvals = await this.prApi.listApprovedReviewsAuthors(true);
```

Comment:
I feel like this mishmash of adding the author and then filtering him out might be confusing/dangerous.

How about we only add him if needed:

```
if (countAuthor) {
  ruleApprovals.push(this.prApi.getAuthor());
}
```

Reply:
Sounds good. Could you look at the new commit to see if it aligns with what you thought?